x-pack/solutions/security/test/cloud_security_posture_api/es_archives/logs_gcp_audit_ecs_only/data.json

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
regarding the archive name. what you mean by `ecs_only`? 
Worth adding a README explaining the differences between the archives and which one fits to which use case